src/ui/components/AppWrapper/AppWrapper.tsx

```diff
@@ -136,7 +135,13 @@ const peerConnectRequestSignChangeHandler = async (
   event: PeerConnectSigningEvent,
   dispatch: ReturnType<typeof useAppDispatch>
 ) => {
-  //TODO: Handle logic for the accept/decline sing request
+  dispatch(
+    setQueueIncomingRequest({
+      id: "peer-connect-signing",
+      signTransaction: event,
+      type: IncomingRequestType.SIGN_TRANSACTION_REQUEST,
```

Comment:
hmm I don't see the `label` being passed here that is expected in the component...

Reply:
It is not required

Reply:
What about this line? https://github.com/cardano-foundation/cf-identity-wallet/blob/5743df913f042bf2aca7254949924d53bb96bae4/src/ui/pages/SidePage/components/IncomingRequest/components/SignTransactionRequest.tsx#L68

Reply:
hmm, ok, I will add it

Reply:
The label is the name of the dApp connection, not from i18n